app/models/harvest.rb

```diff
@@ -9,7 +9,6 @@ class Harvest < ActiveRecord::Base
   belongs_to :plant_part
   belongs_to :planting
 
-  default_scope { order('created_at DESC') }
```

Comment:
The default scopes i've seen in other projects are e.g. `where(deleted: false)` etc. 

Growstuff has a log of query calls in the views too - where I've usually seen them in the controller with order and pagination controlled there.